x/bsonx/bsoncore/tranquil_document_test.go

```diff
@@ -0,0 +1,35 @@
+package bsoncore
+
+import (
+	"bytes"
+	"testing"
+)
+
+var updateDoc []byte = []byte{0x6f, 0x0, 0x0, 0x0, 0x3, 0x71, 0x0, 0x16, 0x0, 0x0, 0x0, 0x7, 0x5f, 0x69, 0x64, 0x0, 0x62, 0x75, 0x73, 0x69, 0x6e, 0x65, 0x73, 0x73, 0x2d, 0x2d, 0x69, 0x64, 0x0, 0x3, 0x75, 0x0, 0x3d, 0x0, 0x0, 0x0, 0x3, 0x24, 0x73, 0x65, 0x74, 0x4f, 0x6e, 0x49, 0x6e, 0x73, 0x65, 0x72, 0x74, 0x0, 0x2a, 0x0, 0x0, 0x0, 0x7, 0x5f, 0x69, 0x64, 0x0, 0x62, 0x75, 0x73, 0x69, 0x6e, 0x65, 0x73, 0x73, 0x2d, 0x2d, 0x69, 0x64, 0x10, 0x73, 0x65, 0x71, 0x75, 0x65, 0x6e, 0x63, 0x65, 0x5f, 0x76, 0x61, 0x6c, 0x75, 0x65, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x8, 0x6d, 0x75, 0x6c, 0x74, 0x69, 0x0, 0x0, 0x8, 0x75, 0x70, 0x73, 0x65, 0x72, 0x74, 0x0, 0x1, 0x0}
```

Comment:
Is it possible to add a comment what this represents? Supposedly, this doc has multiple values for _id. 

BTW, I am OK with the byte array, it would b just helpful to annotate the test case/value with a comment.

Reply:
Yeah I agree, a comment explaining what the bytes represent would be helpful.  Otherwise these tests look like they will do the trick.

Reply:
comment added. It's the bytes of a single-doc, single-field `$setOnInsert` update request